addon/configuration.js

```diff
@@ -3,9 +3,9 @@ import Ember from 'ember';
 const { getWithDefault, typeOf, deprecate } = Ember;
 
 const DEFAULTS = {
-  baseURL:                     '',
-  authenticationRoute:         'login',
-  routeAfterAuthentication:    'index',
+  baseURL: '',
+  authenticationRoute: 'login',
+  routeAfterAuthentication: 'index',
```

Comment:
I actually think aligned hashes improve readability a lot. Not aligning assignments is fine I guess but I'd keep it for hashes.